src/componnets/ShowProducts.js

```diff
@@ -0,0 +1,24 @@
+import { useLoaderData, useNavigate } from "react-router-dom";
+import Card from "../componnets/Card";
+import classes from "./ShowProducts.module.css";
+const ShowProducts = () => {
```

Comment:
How about naming this "`ProductList`"? "Show" is a verb in this case and should be used only for functions that do something, not components.